bentoml/cli/bento_management.py

```diff
@@ -188,26 +199,30 @@ def list_bentos(
     @click.option(
         '-y', '--yes', '--assume-yes', is_flag=True, help='Automatic yes to prompts'
```

Comment:
is the default value True or False?

Reply:
Could you update the help message to be more specific? `Skip confirmation when deleting specific BentoService bundle`

Reply:
Default is False. Make users more aware of the consequences.